spec/nameable/latin_spec.rb

```diff
@@ -1,284 +1,286 @@
-    expect(n.first).to eq('Chris')
-  end
-  it '.middle' do
-    expect(n.middle).to be_nil
-  end
-  it '.last' do
-    expect(n.last).to eq('Horn')
-  end
-  it '.suffix' do
-    expect(n.suffix).to eq('Ph.D.')
+  context('with a simple suffix') do
+    subject(:n) { Nameable::Latin.new.parse("Chris Horn PhD") }
+    it '.prefix' do
+      expect(n.prefix).to be_nil
+    end
+    it '.first' do
+      expect(n.first).to eq('Chris')
+    end
+    it '.middle' do
+      expect(n.middle).to be_nil
+    end
+    it '.last' do
+      expect(n.last).to eq('Horn')
+    end
+    it '.suffix' do
+      expect(n.suffix).to eq('Ph.D.')
+    end
   end
-end
 
-context('prefix patterns') do
-  %w{Mr Mr. Mister}.each do |prefix|
-    it prefix do
-     expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Mr.')
+  context('prefix patterns') do
+    %w{Mr Mr. Mister}.each do |prefix|
+      it prefix do
+       expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Mr.')
+      end
     end
-  end
 
-  %w{Mrs Mrs. Misses}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Mrs.')
+    %w{Mrs Mrs. Misses}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Mrs.')
+      end
     end
-  end
 
-  %w{Ms Ms. Miss}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Ms.')
+    %w{Ms Ms. Miss}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Ms.')
+      end
     end
-  end
 
-  %w{Dr Dr. Doctor}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Dr.')
+    %w{Dr Dr. Doctor}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Dr.')
+      end
     end
-  end
 
-  %w{Rev Rev. Reverand}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Rev.')
+    %w{Rev Rev. Reverand}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Rev.')
+      end
     end
-  end
 
-  %w{Fr Fr. Friar}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Fr.')
+    %w{Fr Fr. Friar}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq('Fr.')
+      end
     end
-  end
 
-  %w{Master Sir}.each do |prefix|
-    it prefix do
-      expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq(prefix)
+    %w{Master Sir}.each do |prefix|
+      it prefix do
+        expect(Nameable::Latin.new.parse("#{prefix} Chris Horn").prefix).to eq(prefix)
+      end
     end
+
   end
 
-end
+  context('suffix patterns (formatted)') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.